scripts/review_gate.py

```
@@ -84,13 +84,28 @@
 
 # 起点より後だが、宣言を持たないことを許すcommit。
 #
-# 対象はPR #160のsquashである。**Gateがrequired status checkでなかった期間に
-# mergeされた。**
 # `AGENTS.md`が履歴書き換えを禁じているため、後からtrailerを付けられない。
+# **免除するのは宣言の記録だけである。**中身はいずれもPull Requestのreviewを通っている。
+#
+# **理由は1つではない。原因ごとに書き分ける。**同じ「後から付けられない」で
+# まとめると、再発を止める手がかりが消える。
+#
+# - `9c91f913`（PR #160）。**Gateがrequired status checkでなかった期間にmergeされた。**
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)